docs/UserGuide.md

```diff
@@ -304,7 +306,8 @@ will search for all items matching `GET` and `POST`.
 
 #### 4.2.6 List all saved API endpoints: <span class="main-command">list</span>
 
-**Description:** Show a list of all API endpoints in the API endpoint list. If there are no endpoints in the API endpoint list, nothing will show up on the Endpoint List bar.
+**Description:** Show a list of all API endpoints in the API endpoint list. If there are no endpoints in the API 
```

Comment:
this can be

 show -> shows

Reply:
This is the format for all commands, I think it might be ok in terms of an imperative verb, like the requirement for Commit message to start with `Update` instead of `Updates`

Reply:
ok makes sense

Reply:
ok thanks, it involves changes to all command so maybe the next person can consider if there is a real need.